whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -2136,109 +2245,329 @@
+                "typeNote": "Digitalt faksimil"
+              }
             }
           }
         ],
         "_spec": [
           {
             "source": {
               "mainEntity": {
-                "seriesMembership": [
+                "@type": "Electronic",
+                "genreForm": [
+                  {"@id": "https://id.kb.se/term/saogf/Digitalt%20faksimil"},
+                  {"@id": "https://id.kb.se/term/saogf/Elektronisk%20text"}
+                ],
+                "production": [
                   {
-                    "@type": "SeriesMembership",
-                    "seriesStatement": "...",
-                    "marc:seriesTracingPolicy": "0",
-                    "inSeries": {"...": "..."}
+                    "@type": "DigitalReproduction"
                   }
                 ]
               }
             },
             "result": "source",
-            "back": "source"
+            "back": {
+              "mainEntity": {
+                "@type": "Electronic",
+                "genreForm": [
+                  {"@id": "https://id.kb.se/term/saogf/Digitalt%20faksimil"},
+                  {"@id": "https://id.kb.se/term/saogf/Elektronisk%20text"}
+                ],
+                "production": [
+                  {
+                    "@type": "DigitalReproduction",
+                    "typeNote": "Digitalt faksimil och elektronisk text"
+                  }
+                ]
+              }
+            }
           },
           {
             "source": {
               "mainEntity": {
-                "seriesMembership": [
+                "@type": "Electronic",
+                "genreForm": [
+                  {"@id": "https://id.kb.se/term/saogf/Digitalt%20faksimil"}
+                ],
+                "production": [
                   {
-                    "@type": "SeriesMembership",
-                    "seriesStatement": "...",
-                    "inSeries": {"...": "..."}
+                    "@type": "Reproduction"
                   }
                 ]
               }
             },
             "result": "source",
             "back": {
               "mainEntity": {
-                "seriesMembership": [
+                "@type": "Electronic",
+                "genreForm": [
+                  {"@id": "https://id.kb.se/term/saogf/Digitalt%20faksimil"}
+                ],
+                "production": [
                   {
-                    "@type": "SeriesMembership",
-                    "seriesStatement": "...",
-                    "marc:seriesTracingPolicy": "1",
-                    "inSeries": {"...": "..."}
+                    "@type": "Reproduction",
+                    "typeNote": "Digitalt faksimil"
                   }
                 ]
               }
             }
+          }
+        ]
+      },
+      {
+        "type": "InjectWhenMatchingOnRevert",
+        "_TODO": "This should be closeMatch/broader-driven (and computed on search)",
```

Comment:
Note that this is a manual mapping which mirrors (one of) the mappings defined in libris/definitions#341

Later on, if those mappings have landed and have a more powerful mechanism of getting at term-matching relationships in core, we could switch to using that here, and remove these hard-coded patterns and term references.